xml/security_ldap.xml

```diff
@@ -398,266 +399,195 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
+       confusion and to enable a better understanding of how it all works.
     </para>
-    <note>
-     <title>Instance Name</title>
-      <para>
-       If not specified otherwise, the default instance name is
-       <literal>localhost</literal>. The instance name cannot
-       be changed after the instance has been created.</para>
-     <para>
-      To check the name of an instance you have not created yourself, use
-      the <command>dsctl -l</command> command.</para>
-    </note>
     <para>
      <xref linkend="ex-ldap-389-ds-inf" xrefstyle="select:label"/> shows an
-     example configuration file that you can use as a starting point. Alternatively,
-     use <command>dscreate create-template</command> to create a template
-     <filename>*.inf</filename> file. The template is commented and pre-filled,
-     so you can adjust its variables to your needs. For more details, see the
-     man page of <command>dscreate</command>.
+     example configuration file that you can use to create a new &ds389; instance.
+     You may copy and use this file, though be sure to create your own password.
     </para>
    <procedure>
     <step>
      <para>
-      If you want to set up a trial instance, start an editor and save the
-      following as <filename>/tmp/instance.inf</filename>:
+      Copy the following example file, <filename>389ds-test.inf</filename>, to 
+      your home directory:
      </para>
      <example xml:id="ex-ldap-389-ds-inf">
-      <title>Basic Instance Configuration File</title>
-      <screen># /tmp/instance.inf
+      <title>Minimal &ds389; Instance Configuration File</title>
+      <screen># <replaceable>389ds-test.inf</replaceable>
+          
 [general]
-config_version = 2
+config_version = 2 <co xml:id="co-ldap-389-ds-config_version"/>
 
 [slapd]
-root_password = <replaceable>YOUR_PASSWORD_FOR_CN=DIRECTORY_MANAGER</replaceable><co xml:id="co-ldap-389-ds-rootpasswd"/>
+root_password = <replaceable>password</replaceable><co xml:id="co-ldap-389-ds-rootpasswd"/>
+self_sign_cert = True <co xml:id="co-ldap-389-ds-cert"/>
+instance_name = <replaceable>389ds-test</replaceable>
 
 [backend-userroot]
-sample_entries = yes
-suffix = dc=example,dc=com</screen>
+sample_entries = yes <co xml:id="co-ldap-389-ds-sample-entries"/>
+suffix = <replaceable>dc=test1,dc=com</replaceable></screen>
+
       <calloutlist>
+       <callout arearefs="co-ldap-389-ds-config_version">
+        <para>
+         This line is required, indicating that this is a version 2 setup .inf
+         file.
+        </para>
+       </callout>
        <callout arearefs="co-ldap-389-ds-rootpasswd">
         <para>
-         Set the <varname>root_password</varname> to the password for
-         the directory server &rootuser; user. The password is used for LDAP
-         server administration only.
+         Create a <varname>root_password</varname> for
+         the directory server &rootuser; user, <literal>dirsrv</literal>. 
```

Comment:
the root_password goes to the ldap user `cn=Directory Manager` user, not `dirsrv` which is the unix process user/owner.

Reply:
cool thx